### ISSUE TITLE
Set room temperature for setup bypass 

### ISSUE THREAD

**Issue by pantherhunter** [laszlojakab/homeassistant-easycontrols#85]
Is there a opportunity to setup the room temperature values vor the bypass setup to control the bypass mode?

**Comment by laszlojakab** [comment]
Do you mean this temperature?

![image](https://user-images.githubusercontent.com/6196882/183151504-ce85bf94-aea9-44a7-b9b2-59e5b4a433f0.png)

**Comment by pantherhunter** [comment]
Exactly. With this temperature settings, the bypass manipulation is possible. There is no other way to set the bypass manually. 

Examplee: If I set it manual to 29 the bypass will close. Under 25 the bypass will open.

Is there a possible to integration this temp setting to your easy controls app? Would be awesome!!!

Greetings Eddy

**Comment by CrimsonAC** [comment]
I would love to see that, too.

It's a shame, that Helios didn't implement a simple parameter for de-/activating the bypass. 
Manipulating this temperature value is the only possible way to set the bypass manually.

Have a nice day.

**Comment by djm193** [comment]
Activate and deactivate the baypass would be really nice! By now I did not find out how to handle this. or is it currently possible to open/close bypass manually?

**Comment by pycolas** [comment]
Hi, i saw you add somes bypass vars for this (VARIABLE_BYPASS_EXTRACT_AIR_TEMPERATURE, VARIABLE_BYPASS_OUTDOOR_AIR_TEMPERATURE) in e7f35a7.
Do you plan to implement this feature ? Do you need some help ?

**Comment by laszlojakab** [comment]
@pycolas yes, I plan to implement these variables (instead of a bypass switch). I have started it in 2023, but then I didn't have time to finish/test it. The code I started writing back then is here: 
[`develop...feat/add-by-pass-number-entities`#diff-921e374486](https://github.com/laszlojakab/homeassistant-easycontrols/compare/develop...feat/add-by-pass-number-entities#diff-921e37448652063c4d59a320939590871e138bbc1f46f062e4374ca892663adc). Unfortunately, I can't even remember where I left off, what was completed, what works, and what doesn't (glancing at the WIP commit, there is a lot of junk and unfinished stuff in the code). Maybe the code is deprecated (I remember some changes with NumberEntity in HomeAssistant).

If you have time to implement the `EasyControlsNumberEntity` it would be great. Then adding these variables (and maybe others) would be piece of cake.

**Comment by Murmillox** [comment]
@laszlojakab
I needed the "NumberEntity" to run. How should I best send you the data, so that you have as little work with it as possible?
I changed the following files:
- __init__.py
- number.py
- coordinator.py
- const.py

Not used:
- eazyctrl.py

I no longer have any errors in the LOG, apart from what is mentioned in point #116.

**Comment by laszlojakab** [comment]
@Murmillox can you open a pull request with the modifications?

**Comment by Murmillox** [comment]
@laszlojakab 
I got the following error message when creating a pull request:
"Pull request creation failed. Validation failed: must be a collaborator"
What is my mistake? Or do you need to adjust something else?

**Comment by Murmillox** [comment]
Hello, I wanted to ask about my pull request. Has anyone had time to look at it?

Kind regards 
Murmillo

**Comment by mfenske** [comment]
@Murmillox Thanks, I'll switch to your fork until the pull request gets merged. I was already seeing myself wasting the whole weekend on Modbus.

**Comment by Sebastien-Gastard** [comment]
I've just write a little piece of code to do this (I modify the limit for the temperature outside)

In your configuration.yaml:
`shell_command:
  helios_bypass_enable: >
    python3 -c "
    from eazyctrl import EazyController;
    ctrl = EazyController('192.168.X.X');
    ctrl.set_variable('v01036', '14')
    "
  helios_bypass_disable: >
    python3 -c "
    from eazyctrl import EazyController;
    ctrl = EazyController('192.168.X.X');
    ctrl.set_variable('v01036', '50')
    "
  helios_bypass_reset: >
    python3 -c "
    from eazyctrl import EazyController;
    ctrl = EazyController('192.168.X.X');
    ctrl.set_variable('v01036', '14')
    "`

And for automations.yaml:
`- id: switch_bypass_vmc
  alias: "VMC Bypass - Control temperature"
  description: "active the bypass only if T° ext < T° int - 2°, between may and september, check every 3 hours"
  trigger:
    - trigger: time_pattern
      hours: "/3"
  condition:
    - condition: template
      value_template: >
        {{ states('sensor.helioskwl_outside_air_temperature') not in ['unknown', 'unavailable']
            and states('sensor.helioskwl_extract_air_temperature') not in ['unknown', 'unavailable'] }}
    - condition: template
      value_template: >
        {{ now().month >= 5 and now().month <= 9 }}
  action:
    - choose:
        - conditions:
            - condition: template
              value_template: >
                {{ states('sensor.helioskwl_outside_air_temperature') | float
                    < (states('sensor.helioskwl_extract_air_temperature') | float - 2) }}
          sequence:
            - action: shell_command.helios_bypass_enable
        - conditions:
            - condition: template
              value_template: >
                {{ states('sensor.helioskwl_outside_air_temperature') | float
                    >= (states('sensor.helioskwl_extract_air_temperature') | float - 2) }}
          sequence:
            - action: shell_command.helios_bypass_disable
  mode: single

- id: reinit_vmc
  alias: "VMC Bypass - Reset setup"
  description: "Reset limit october 1st"
  trigger:
    - trigger: time
      at: "00:01:00"
  condition:
    - condition: template
      value_template: >
        {{ now().month == 10 and now().day == 1 }}
  action:
    - action: shell_command.helios_bypass_reset
  mode: single`
